docs/development/documentation_preview.rst

```diff
@@ -4,11 +4,12 @@
 Documentation Preview
 *********************
 
-Most of the time it's enough to build the documentation locally and see how things are going. But sometimes 
-it's nice to share our changes and get some feedback from other collaborators. 
+To preview your changes to the documentation please:
 
-To preview your changes to the documentation first make sure you have GitHub Actions activated on your fork
-(see the Actions tab). Then, there are two ways to trigger the build:
+#. Enable GitHub Actions to run on your fork (see the *Actions* tab).
+#. Make sure your fork is deploying GitHub Pages inside the root of ``gh-pages`` branch (see the *Settings* tab).
```

Comment:
You may want to mention that one needs to go to Settings --> Pages

Reply:
Ok.

Reply:
Done.